adal/devApps/XFormsApp.iOS/AppDelegate.cs

```diff
@@ -52,6 +52,9 @@ public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsAppli
         public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
             global::Xamarin.Forms.Forms.Init();
+            #if ENABLE_TEST_CLOUD
```

Comment:
>ENABLE_TEST_CLOUD [](start = 16, length = 17)

when is this set?

Reply:
This is set externally by the build definition